Formula/kyuubi.rb

```diff
@@ -0,0 +1,36 @@
+class Kyuubi < Formula
+  desc "Serverless SQL on data warehouses and lakehouses"
+  homepage "https://kyuubi.apache.org/"
+  version "1.6.1-incubating"
+  url "https://dlcdn.apache.org/kyuubi/kyuubi-#{version}/apache-kyuubi-#{version}-bin.tgz"
+  sha256 "a64a6e5d71b39b0e494597282493250df1c74714fc2bba5057946926c0a32ba2"
+  license "Apache-2.0"
+  head "https://github.com/apache/kyuubi.git", branch: "master"
+
+  depends_on "openjdk@8"
```

Comment:
Does this work with the other jdk? If yes, we might want to do something like this

```
JAVA_HOME:     Language::Java.overridable_java_home_env("1.8")[:JAVA_HOME],
```